import-export-cli/integration/apim/apim.go

```diff
@@ -2039,6 +2049,221 @@ func (instance *Client) registerClient(username string, password string) dcrResp
 	return jsonResp
 }
 
+// AddThrottlePolicy : Add new Throttle Policy of different policy types to APIM
+func (instance *Client) AddThrottlePolicy(t *testing.T, policy interface{}, policyType string) interface{} {
+	var throttlePolicyResponse interface{}
+
+	throttlePolicyURL := instance.adminRestURL + "/throttling/policies/" + policyType
+
+	data, err := json.Marshal(policy)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	request := base.CreatePost(throttlePolicyURL, bytes.NewBuffer(data))
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.AddThrottlePolicy()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.AddThrottlePolicy()", response, 201)
+
+	json.NewDecoder(response.Body).Decode(&throttlePolicyResponse)
+
+	return throttlePolicyResponse
```

Comment:
Please add a t.Cleanup here as well. (Make sure to add a doClean option as well as in [1], because in the update situation, you need to make doClean false)

Refer [1].

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/integration/apim/apim.go#L511

[1]

Reply:
fixed with 53f8cc8609cb4711d71c229466ebbd0f9c8f10e5

Reply:
Fixed with 3c4e0a6955b4c9779a67ea315b574412c7967a6d